src/ui/components/ObjectView.vue

```diff
@@ -185,17 +188,17 @@ export default {
 
             let objectPath = this.currentObjectPath || this.objectPath;
 
-            if (provider.edit && this.showEditView) {
+            if (provider.edit && this.options.showEditView) {
                 if (this.openmct.editor.isEditing()) {
-                    this.currentView = provider.edit(this.domainObject, true, objectPath);
+                    this.currentView = provider.edit(this.domainObject, true, objectPath, this.options);
```

Comment:
These options are just more API that we have to document and maintain. 

I also think maybe we have it backwards, instead of the timeline telling the plot how to render itself, I think views will have to be aware of the context they're in, and adjust their own rendering accordingly. ie. plots must have some awareness that they're in a timeline.

We need proper API for this, but we don't need to solve it right now. We can defer until we integrate more views into timelines and have a better idea where the commonalities are.

For now it would be fine if the plot view provider just looked at the object path and figured out that it's inside a timeline, and adjusted itself accordingly. I don't want this stuff manifesting in our public API as an opaque `options` object with some random specific styling instructions in it.

Reply:
Options have been removed and props flattened